### ISSUE TITLE
Can't install from stable PPA in Ubuntu 19.04

### ISSUE THREAD

**Issue by GrantM11235** [minetest/minetest#8647]
##### Minetest version

```
5.0.1-ppa0~ubuntu19.04.1
```

##### OS / Hardware

Operating system: Ubuntu 19.04
CPU: amd64

##### Summary

After adding the stable PPA, I can't upgrade minetest
```
$ sudo apt upgrade minetest
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Calculating upgrade... Done
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 minetest : Depends: libleveldb1v5 but it is not installable
E: Broken packages
```
It looks like the [libleveldb1v5](https://packages.ubuntu.com/cosmic/libleveldb1v5) package was renamed to [libleveldb1d](https://packages.ubuntu.com/disco/libleveldb1d)

##### Steps to reproduce

In Ubuntu 19.04, add the stable PPA and try to install or upgrade minetest

**Comment by style-nine** [comment]
https://launchpad.net/~minetestdevs/+archive/ubuntu/stable/+packages

> Depends: libc6 (>= 2.29), libcurl3-gnutls (>= 7.16.2), libfreetype6 (>= 2.2.1), libgcc1 (>= 1:3.0), libirrlicht1.8, libleveldb1v5, libluajit-5.1-2 (>= 2.0.4+dfsg), libncursesw6 (>= 6), libopenal1 (>= 1.14), libsqlite3-0 (>= 3.5.9), libstdc++6 (>= 6), libtinfo6 (>= 6), libvorbisfile3 (>= 1.2.0), libx11-6, zlib1g (>= 1:1.1.4)

**Comment by tenplus1** [comment]
I had to download the libleveldb1v5 .deb file and install it before installing Minetest dev.

**Comment by nerzhul** [comment]
i like when packages are renamed haha

**Comment by HybridDog** [comment]
I think it is possible to specify alternatives for a specific package dependency: https://askubuntu.com/a/940177

**Comment by GrantM11235** [comment]
It looks like this was fixed in 5.0.1-ppa1
